eng-paths/sr_staff_engineer.md

```diff
@@ -122,14 +124,15 @@ Sr Staff Engineer
 </td></tr>
 <tr><td markdown="1" style="font-weight:bold">
 
-- Leads cross-teams projects and initiatives, and inter-team communication.
+- Leads cross-teams projects and initiatives, and inter-team and company-wide communication.
 - Identifies org-level opportunities, prioritizes, plans, and creates buy-in for resolution.
 - Mentors engineers and fosters their growth.
 - Coaches teams to continuously improve delivery pace and quality.
 - Can step in as a contributor on any team when needed.
 - Can collaborate with internal and external stakeholders in any department, team, or client when technical presence is required.
 - Acts as an external facing advocate and technical authority for Beyond Engineering, drives talent acquisition.
-- Lead critical outages and high-impact issues meetings to ensure the solution taking place is the right one.
+- Leads critical outages and high-impact issues meetings to ensure the solution taking place is the right one.
+
```

Comment:
Not sure, but I think you need to remove the spaces at the EOF on the MD (there are others like this :) )